packages/types/src/Message.ts

```diff
@@ -348,3 +312,44 @@ export type CompressedMessageBody =
   | CompressedSubmitAcceptDelegation
   | CompressedRejectAcceptDelegation
   | CompressedInformCreateDelegation
+
+/**
+ * - `body` - The body of the message, see [[MessageBody]].
+ * - `createdAt` - The timestamp of the message construction.
+ * - `receiverAddress` - The public SS58 address of the receiver.
+ * - `senderAddress` - The public SS58 address of the sender.
+ * - `senderBoxPublicKex` - The public encryption key of the sender.
+ * - `messageId` - The message id.
+ * - `inReplyTo` - The id of the parent-message.
+ * - `references` - The references or the in-reply-to of the parent-message followed by the message-id of the parent-message.
+ */
+export interface IMessage {
```

Comment:
Makes sense to put it last

Reply:
The thing is that there's a lint rule against using types not yet declared, and `IEncryptedMessageContents` right below `IMessage` refers to `IMessage`. If you think this rule is not relevant, I could either ignore it in the lint config or just for this. But otherwise, I would leave `IMessage` where it is right now.

Reply:
No, I think it is good